gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -65,8 +65,8 @@
 
     disposeOnClear(
         addFeatureClicks
-            .switchMapSingle(Result.wrapErrors(dataRepository::addFeature))
-            .subscribe(Result.unwrapErrors(this::showFeatureSheet, this::onAddFeatureError)));
+            .switchMapSingle(Result.mapSingle(dataRepository::addFeature))
+            .subscribe(Result.unwrap(this::showFeatureSheet, this::onAddFeatureError)));
```

Comment:
Please use stacked PRs when there's overlap to avoid redundant diffs.

Reply:
gotcha